webviz_subsurface/plugins/_well_completions/_plugin.py

```diff
@@ -2,12 +2,18 @@
 
 from dash import Dash, html
 from webviz_config import WebvizPluginABC, WebvizSettings
+from webviz_config.deprecation_decorators import deprecated_plugin
 
 from ._business_logic import WellCompletionsDataModel
 from ._callbacks import plugin_callbacks
 from ._layout import layout_tour_steps, main_layout
 
 
+@deprecated_plugin(
+    "This plugin has been replaced by the `WellCompletion` plugin "
```

Comment:
Should we emphasize that the difference (wrt. plugin name) is the removed `s`? Might be hard to see the difference immediately for the users.

Reply:
Good idea :)